stage1/init/pod.go

```diff
@@ -115,9 +115,7 @@ func quoteExec(exec []string) string {
 	}
 
 	var qexec []string
-	qexec = append(qexec, exec[0])
-	// FIXME(vc): systemd gets angry if qexec[0] is quoted
-	// https://bugs.freedesktop.org/show_bug.cgi?id=86171
+	qexec = append(qexec, `"`+exec[0]+`"`)
 
 	if len(exec) > 1 {
 		for _, arg := range exec[1:] {
```

Comment:
Well, we at least need to escape the quotes which may occur in exec[0].  I'm unsure about the other things, exec[0] does seem to be exceptional in systemd, for example:

>       To pass a literal dollar sign, use "$$". Variables whose value is not known at
>       expansion time are treated as empty strings. Note that the first argument (i.e. the
>       program to execute) may not be a variable.

So... yeah.

It would be nice if we could just drop the len(exec) condition, and loop uniformly over exec[], but I don't see why doubling $'s up would be appropriate for exec[0] if exec[0] doesn't support variable substitutions, for example.